libs/ui/src/lib/treemap-chart/treemap-chart.component.ts

```diff
@@ -155,13 +155,13 @@ export class GfTreemapChartComponent
             formatter(ctx) {
```

Comment:
To keep the font sizes as before, you need to adapt the `font` setting (one line above) accordingly 🙂

Reply:
@dtslvr  done sir

Reply:
As the label now only consists of two parts (without symbol), you need to remove the item at the correct position:
```
font: [{ size: 14 }, { lineHeight: 2, size: 14 }],
```

Otherwise it would not work with the `lineHeight` anymore.